executor/common_linux.h

```diff
@@ -5239,3 +5239,44 @@ static long syz_80211_join_ibss(volatile long a0, volatile long a1, volatile lon
 }
 
 #endif
+
+#if SYZ_EXECUTOR || __NR_syz_clone3
+#include <linux/sched.h>
+#include <sched.h>
+
+#define USLEEP_FORKED_CHILD 250000 // 250 ms.
+static long syz_clone3(volatile long a0, volatile long a1)
+{
+#ifdef __NR_clone3
```

Comment:
This will not work for gVisor that doesn't support clone3, because test programs are compiled on a VM file system that  can have any kernel. We can switch to the clone syscall if clone3 returns ENOSYS.

Reply:
This is a not yet tested draft, but thanks for noting the problem early :) 

I'll switch to the clone3 detection in the pkg/host.